client/src/components/Notices/NoticeList.js

```diff
@@ -49,8 +49,8 @@ const NoticeList = ({
   updateValue,
 }: NoticeListProps) => {
   const items =
-    dateSortedNotices.length > 0
-      ? groupBy(dateSortedNotices, (item) => `${item.bulletin_number}/${item.bulletin_year}`)
+    notices.length > 0
```

Comment:
This check isn't needed, is it? It's a bit strange that `items` can be either object or array (but only empty one). Lodash's map should be ok with empty object and I think it should be consistent.